terraform/app/variables.tf

```
@@ -202,6 +208,10 @@ locals {
     {
       name  = "MAVIS__SPLUNK__ENABLED"
       value = var.enable_splunk ? "true" : "false"
+    },
+    {
+      name  = "MAVIS_APP_VERSION" # <-- ADD THIS
```

```suggestion
      name  = "APP_VERSION" # <-- ADD THIS
```

The only reason we have `MAVIS__` prefixed on the others is due to the way we load in the `Settings`. In general I think we can avoid the `MAVIS` prefix until it's strictly necessary.